impl/src/main/java/org/jboss/cdi/tck/tests/definition/stereotype/priority/DeltaAlt2.java

```diff
@@ -0,0 +1,14 @@
+package org.jboss.cdi.tck.tests.definition.stereotype.priority;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.enterprise.inject.Alternative;
+
+@Dependent
+@Alternative
+@AnotherPriorityStereotype // this will give it @Priority(200)
```

Comment:
I know, I chose the latter and this test should reflect it.
And knowing that it can be contentious is the very reason I wanted to add this test to TCKs - to have at least some ground for it if it is reviewed at some later point.